### PR TITLE
doc aws secretsmanager, lookup tables 

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - doc AWS Secrets Manager set-up
 - doc lookup table building for GCP, AWS

### Change implications

 - dependencies added/changed? **no**
 - something important to note in future release notes? **no**

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207658906811813

### CODE REVIEW

File: docs/aws/guides/lookup-tables.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        "manager_id",
+        "manager_email",
+        "join_date",
+        "leave_date"
+      ]
+      columnsToPseudonymize = [
+        "employee_id", # primary key
+      ]
+      columnsToDuplicate   = {
+        "employee_id" = "employee_id_orig"
```
Comment (jlorper) [inline docs/aws/guides/lookup-tables.md:37]:
```suggestion
        "employee_id" = "employee_id_raw"
```
I'd change the name to suggest is not pseudonimized somehow.

**Review by jlorper — APPROVED** [review]

**Review by aperez-worklytics — APPROVED** [review]

File: docs/gcp/guides/lookup-tables.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# Lookup Tables with GCP
```
Comment (davidfq) [inline docs/gcp/guides/lookup-tables.md:1]: This one is not included in the `SUMMARY.md`, on purpose?

File: docs/aws/guides/lookup-tables.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# Lookup Tables with AWS
+
+If you use psoxy to send pseudonymized data to Worklytics and later wish to re-identify the data
```
Comment (davidfq) [inline docs/aws/guides/lookup-tables.md:3]:
```suggestion
If you use the Psoxy to send pseudonymized data to Worklytics and later wish to re-identify the data
```

File: docs/aws/guides/lookup-tables.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+that you export from Worklytics to your premises, you'll need a lookup table in your data warehouse
+to JOIN with that data.
+
+![AWS data flow](lookup-table-data-flow.png)
```
Comment (davidfq) [inline docs/aws/guides/lookup-tables.md:7]: picky detail about image contents: it has too much whitespace below the actual diagram... it's going to create a bad looking result

**Review by davidfq — APPROVED** [review]

File: docs/aws/guides/lookup-tables.md (hunk 1 above)
Comment (eschultink) [reply docs/aws/guides/lookup-tables.md:37]: but that's what original means ... it's the original value, before transformation

**Pull request merged**